.github/workflows/linux-eic-shell.yml

```diff
@@ -929,6 +929,7 @@ jobs:
           export JANA_PLUGIN_PATH=$PWD/install/lib/EICrecon/plugins:/usr/local/plugins
           prmon --json-summary rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.prmon.json -- \
           $PWD/install/bin/eicrecon ${{env.JANA_OPTIONS}} ${{matrix.nthreads > 1 && format('-Pnthreads={0}', matrix.nthreads) || ''}} -Ppodio:output_file=rec_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4eic.root sim_dis_${{matrix.beam}}_minQ2=${{matrix.minq2}}_${{ matrix.detector_config }}.edm4hep.root -Pacts:WriteObj=true -Pacts:WritePly=true -Pplugins=janadot,janatop $(<${{ github.workspace }}/.github/janadot.groups)
+      continue-on-error: ${{matrix.nthreads > 1}}
```

Comment:
Looks like it will ignore runtime errors for mt running. Why do we want that?

Reply:
This was intended for the initial phases of roll-out. I would prefer we first get some more gentle notifications of breakage rather than ending up with broken CI on main due to some algorithm that is thread-unsafe but that hasn't shown up yet in this branch.

Reply:
If you error and continue in eicrecon, it will not produce an artifact, so `compare-single-multi-threaded` will not be able to download it. That was my thinking.